### ISSUE TITLE
6 links don't take item level into acount for price

### ISSUE THREAD

**Issue by Immow** [SnosMe/awakened-poe-trade#9]
When checking the price of a 6 link item, itemlevel is an important factor. For example my bow that I found (Citadel Bow ilv 84) is shown as  30 ~ 50 chaos. But if you search it with proper ilvl settings it's around 1.5 to 4 ex

**Comment by SnosMe** [comment]
Can you share Item info (Ctrl+C) here

**Comment by Immow** [comment]
Rarity: Normal
Citadel Bow
--------
Bow
Physical Damage: 25-101
Critical Strike Chance: 6.00%
Attacks per Second: 1.25
--------
Requirements:
Level: 58
Dex: 185
--------
Sockets: G-G-G-G-G-G 
--------
Item Level: 84
--------
Note: ~price 1.5 exa

**Comment by nandotech** [comment]
Another example of this.

My Belly of the Beast shows as 5c estimate even though I upgraded it from 1s to 6s then 6 link. The ML prediction is same 5c, though the price list below accurately shows listings starting at 3ex.

```
Rarity: Unique
Belly of the Beast
Full Wyrmscale
--------
Armour: 686 (augmented)
Evasion Rating: 254
--------
Requirements:
Level: 46
Str: 68
Dex: 68
--------
Sockets: R-G-R-R-G-R 
--------
Item Level: 80
--------
170% increased Armour
37% increased maximum Life
+13% to all Elemental Resistances
50% increased Flask Life Recovery rate
Extra gore
--------
There is no safer place
Than the Belly of the Beast
```

**Comment by SnosMe** [comment]
Fixed by 37de76553f701ceb7dde287d63b79498b6d0bdc0

![image](https://user-images.githubusercontent.com/4292308/72225128-7604b380-358a-11ea-9a02-7cead2ad34b8.png)
